internal/daemon/worker/worker.go

```diff
@@ -75,7 +75,7 @@ type Worker struct {
 	updateTags *ua.Bool
 
 	// The storage for node enrollment
-	WorkerAuthStorage             nodeenrollment.Storage
+	WorkerAuthStorage             *nodeefile.FileStorage
 	WorkerAuthCurrentKeyId        string
 	WorkerAuthRegistrationRequest string
 	workerAuthSplitListener       *splitlistener.SplitListener
```

Comment:
I know, it's out of scope of this PR, but can we fix this studder?  perhaps change the package name to `net`?

Reply:
Yeah -- I'll do that and merge in here before I merge this

Reply:
Done